model/src/main/kotlin/VcsInfo.kt

```diff
@@ -46,12 +47,19 @@ data class VcsInfo(
          */
         val revision: String,
 
+        /**
+         * The VCS-specific revision resolved during downloading from the VCS. In contrast to [revision] this must not
+         * contain symbolic names like branches or tags.
+         */
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val resolvedRevision: String? = null,
```

Comment:
Thinking about it again: Why is this nullable? We can always resolve the revision, even if it's then sometimes just the same as the (original) revision. So why should we make it optional?

Reply:
This would require a big change in behaviour, currently we do not query any VCS repositories in the analyzer.